bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -93,7 +97,9 @@ spec:
               - -c
               - |
                 retries=0
-                while ! ovn-nbctl -t 5 set-connection ptcp:{{.OVN_NB_PORT}} -- set connection . inactivity_probe=0; do
+                while ! ( ovn-nbctl set-ssl /ovn-cert/tls.key /ovn-cert/tls.crt /ovn-ca/ca-bundle.crt && \
```

Comment:
@squeed instead of `set-ssl`, why aren't we passing the SSL options to ovn-ctl in the first place?

```
  --ovn-nb-db-ssl-key=KEY OVN Northbound DB SSL private key file
  --ovn-nb-db-ssl-cert=CERT OVN Northbound DB SSL certificate file
  --ovn-nb-db-ssl-ca-cert=CERT OVN Northbound DB SSL CA certificate file
  --ovn-sb-db-ssl-key=KEY OVN Southbound DB SSL private key file
  --ovn-sb-db-ssl-cert=CERT OVN Southbound DB SSL certificate file
  --ovn-sb-db-ssl-ca-cert=CERT OVN Southbound DB SSL CA certificate file
```

Reply:
As if that worked. It doesn't.

Reply:
@squeed not as far as I can tell with `set -x` at the top of the ovn-ctl script. They get passed to the database process.

Reply:
This totally works. Duh. Fixed.